tests/qos/qos_sai_base.py

```diff
@@ -601,10 +622,12 @@ def dutConfig(
             testPortIds = dualTorPortIndexes
 
         testPorts = self.__buildTestPorts(request, testPortIds, testPortIps, src_port_ids, dst_port_ids)
+        dutinterfaces = {}
+        for port, index in mgFacts["minigraph_ptf_indices"].items():
+            if 'Ethernet-Rec' not in port and 'Ethernet-IB' not in port:
+                dutinterfaces[index] = port
         yield {
-            "dutInterfaces" : {
```

Comment:
Tested this change. It is failing for me. If i revert the code to original then it's ok. please check. I am running "qos/test_qos_sai.py::TestQosSai::testParameter" test case.

Reply:
This code change is not allowing dutConfig["dutInterfaces"] to be populated.

Reply:
Also please update "rand_one_dut_hostname" with "enum_rand_one_per_hwsku_frontend_hostname" in all other places as it is failing tests.

Reply:
@vmittal-msft  There is no 'rand_one_dut_hostme' in the qos files that we have touched in this PR.

Reply:
With regards to dutinterfaces, can you please try this simple test and see what the difference is in the two dicts dutinterfaces and dutinterfaces_orig

```
def test_mg_facts_ptf_index(duthosts, enum_rand_one_per_hwsku_frontend_hostname, tbinfo):
    duthost = duthosts[enum_rand_one_per_hwsku_frontend_hostname]
    mgFacts = duthost.get_extended_minigraph_facts(tbinfo)
    dutinterfaces = {}
    for port, index in mgFacts["minigraph_ptf_indices"].items():
        if 'Ethernet-Rec' not in port and 'Ethernet-IB' not in port:
            dutinterfaces[index] = port

    dutinterfaces_orig = {index: port for port, index in mgFacts["minigraph_ptf_indices"].items()}
```

Reply:
they are different. 
1st o/p - 64: u'Ethernet0', 65: u'Ethernet4'
2nd o/p - 64: u'Ethernet-IB0', 65: u'Ethernet-Rec0'
2nd o/p doesn't have Ethernet0/4

Reply:
This is working fine for me.